### PR TITLE
LPS-143632 Remove LanguageUtil usages in portal module

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
Could you please help me scan this test result?
Thank you!

**Comment by catarineaguiar** [comment]
ci:test:acceptance

File: modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java [hunk 1]
```diff
@@ -75,14 +75,11 @@ public void processInactiveRequest(
...
+			_portal.getLocale(httpServletRequest), messageKey,
+			StringPool.BLANK);
 
-		String message = null;
-
-		if (LanguageUtil.isValidLanguageKey(locale, messageKey)) {
-			message = LanguageUtil.get(locale, messageKey);
-		}
-		else {
+		if (Validator.isNull(message)) {
```
Comment (tinatian) [inline modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java:82]: @julschong Here, it is not a simple replacing LanguageUtil with _language,it is kind of risky, suppose you have have something wrong with this commit, you have to figure out the reason in changes for 60 files.
Please do simple replacement in this commit and you and add another commit to make other changes, in this case, this kind of change is highlighted.

Tina.

**Comment by tinatian** [comment]
@julschong Please see my comment, and review your other changes one more time, and resend.

Thanks.
Tina

File: modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java (hunk 1 above)
Comment (julschong) [reply modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java:82]: @tinatian 
will separate this change in a second commit.

**Pull request closed without merging**